docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -3,36 +3,18 @@ title: Use Teleport for Application Access
 description: How to configure Teleport for Application Access.
 ---
 
```

Comment:
I think it would be helpful to add an introductory sentence here. For now, this could be something like:

```suggestion
In this guide, we will show you how to register a web application with your 
Teleport cluster in order to access it securely.
```